.gitignore

```diff
@@ -2,10 +2,13 @@ dist/
 dist-newstyle/
 .ghc.environment.*
 ghc/
+ghc-lib-parser/
+ghc-lib/
```

Comment:
Am I correct that it should have always been like this and this is just fixing the situation?

Reply:
Yes, that's right.